README.md

```diff
@@ -26,339 +26,211 @@
+  [TensorRT](https://github.com/triton-inference-server/tensorrt_backend), 
+  [TensorFlow](https://github.com/triton-inference-server/tensorflow_backend), 
+  [PyTorch](https://github.com/triton-inference-server/pytorch_backend), 
+  [ONNX](https://github.com/triton-inference-server/onnxruntime_backend), 
+  [OpenVINO](https://github.com/triton-inference-server/openvino_backend), 
+  [Python](https://github.com/triton-inference-server/python_backend), and more
+- Learn how to [optimize performance](docs/optimization.md) using the 
+  [Performance Analyzer](docs/perf_analyzer.md) and 
+  [Model Analyzer](https://github.com/triton-inference-server/model_analyzer)
+- Learn how to [manage loading and unloading models](docs/model_management.md) in 
+  Triton
+- Send requests directly to Triton with the [HTTP/REST JSON-based
+  or gRPC protocols](docs/inference_protocols.md#httprest-and-grpc-protocols)
+
+#### Client Support and Examples
+
+A Triton *client* application sends inference and other requests to Triton. The 
+[Python and C++ client libraries](https://github.com/triton-inference-server/client)
+provide APIs to simplify this communication.
+
+- Review client examples for [C++](https://github.com/triton-inference-server/client/blob/main/src/c%2B%2B/examples),
+  [Python](https://github.com/triton-inference-server/client/blob/main/src/python/examples),
+  and [Java](https://github.com/triton-inference-server/client/blob/main/src/java/src/main/java/triton/client/examples)
+- Configure [HTTP](https://github.com/triton-inference-server/client#http-options)
+  and [gRPC](https://github.com/triton-inference-server/client#grpc-options)
+  client options
+- Send input data (e.g. a jpeg image) directly to Triton in the [body of an HTTP 
+  request without any additional metadata](https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_binary_data.md#raw-binary-request)
+
+### Extend Triton
+
+[Triton Inference Server's archicture](docs/architecture.md) is specifically 
+designed for modularity and flexibility
+
+- [Customize Triton Inference Server](docs/compose.md) for your use case
+- [Create custom backends](https://github.com/triton-inference-server/backend)
+  in either [C/C++](https://github.com/triton-inference-server/backend/blob/main/README.md#triton-backend-api)
+  or [Python](https://github.com/triton-inference-server/python_backend)
+- Create [decouple backends and models](docs/decoupled_models.md) that can send 
+  multiple responses for a request or not send any responses for a request
+- Use a [Triton repository agent](docs/repository_agents.md) to add functionality
+  that operates when a model is loaded and unloaded, such as authentication, 
+  decryption, or conversion
+- Deploy Triton on [Jetson and JetPack](docs/jetson.md)
+- [Use Triton on AWS 
+   Inferentia](https://github.com/triton-inference-server/python_backend/tree/main/inferentia)
+
+### Additional Documentation
```

Comment:
Should mention `docs/README.md` for a complete list of Triton features.

Reply:
No need, it is called out in the `User Guide` and `Developers Guide` specifically.